test/uk/gov/hmrc/selfassessmentrefundfrontend/controllers/refundRequestJourney/SelectRepaymentAmountControllerSpec.scala

```diff
@@ -60,7 +60,7 @@ class SelectRepaymentAmountControllerSpec
   private val selectAmountPageHeadingWelsh = "Faint o ad-daliad yr hoffech ei gael?"
 
   "GET /refund-amount" when {
-    val fakeRequest = FakeRequest("GET", "/self-assessment-refund-frontend/refund-amount")
+    val fakeRequest = FakeRequest("GET", "/request-a-self-assessment-refund/refund-amount")
```

Comment:
You didn't change anything around urls for this change right? (Or is it just a clarity thing)

Reply:
No I haven't changed anything around url's, this should have always been "/request-a-self-assessment-refund/refund-amount" in the test, as refund-amount page is under request-a-self-assessment-refund - so corrected it